specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2020-12-01-preview/confidentialledger.json

```diff
@@ -512,13 +512,47 @@
         "modelAsString": true
       }
     },
-    "ConfidentialLedgerCertUser": {
+    "LedgerPermission": {
+      "description": "Permission associated with the Security Principal of Ledger",
+      "type": "string",
+      "enum": [
+        "Unknown",
```

Comment:
Is `Unknown` necessary? How would it be different from not specifying a permission at all?

Reply:
Yeah actually here it doesn't make sense to have unknown. Will fix that